crawler/attribution/entrypoints/eval.js

```diff
@@ -0,0 +1,6 @@
+function doEval () {
+    // eslint-disable-next-line no-eval
+    eval(`console.log("${document.currentScript.src}", navigator.userAgent)`);
```

Comment:
Should we test `new Function('....')` also?

Reply:
good idea, I've added a test